payments/stripe/__init__.py

```diff
@@ -76,3 +90,146 @@ class StripeCardProvider(StripeProvider):
     """
 
     form_class = PaymentForm
+
+
+@dataclass
+class StripeProductData:
+    name: str
+    description: Optional[str] = field(init=False, repr=False, default=None)
+    images: Optional[str] = field(init=False, repr=False, default=None)
+    metadata: Optional[dict] = field(init=False, repr=False, default=None)
+    tax_code: Optional[str] = field(init=False, repr=False, default=None)
+
+
+@dataclass
+class StripePriceData:
+    currency: str
+    product_data: StripeProductData
+    unit_amount: int
+    recurring: Optional[dict] = field(init=False, repr=False, default=None)
+    tax_behavior: Optional[str] = field(init=False, repr=False, default=None)
+
+
+@dataclass
+class StripeLineItem:
+    price_data: StripePriceData
+    quantity: int
+    adjustable_quantity: Optional[dict] = field(init=False, repr=False, default=None)
+    dynamic_tax_rates: Optional[dict] = field(init=False, repr=False, default=None)
+    tax_rates: Optional[str] = field(init=False, repr=False, default=None)
+
+
+class StripeProviderV3(BasicProvider):
+    """Provider backend using `Stripe <https://stripe.com/>` api version 3_.
+
+    This backend does not support fraud detection.
+
+    :param secret_key: Secret key assigned by Stripe.
+    :param payment_method_types: From Stripe API, comma separated
+    :param use_token: Use instance.token instead of instance.pk in client_reference_id
+    """
+
+    form_class = PaymentFormV3
+    session = None
+
+    def __init__(
+        self,
+        secret_key,
```

Comment:
Given that stripe calls this `api_key`, it would be clearer to keep the same naming here.